src/app/my-blog/components/layout/layout.component.html

```diff
@@ -0,0 +1,12 @@
+<main class="layout">
+  <div class="layout__blog-list">
+    <ul>
+      <ng-container *ngFor="let post of posts">
```

Comment:
<app-blog-item *ngFor="let post of posts" [post]="post"></app-blog-item>
No need of ng-container

Reply:
fixed